src/renderer/components/general-settings/general-settings.js

```diff
@@ -163,11 +173,13 @@ export default defineComponent({
       if (process.env.SUPPORTS_LOCAL_API) {
         return [
           this.$t('Settings.General Settings.Preferred API Backend.Invidious API'),
-          this.$t('Settings.General Settings.Preferred API Backend.Local API')
+          this.$t('Settings.General Settings.Preferred API Backend.Local API'),
+          this.$t('Settings.General Settings.Preferred API Backend.Piped API')
         ]
       } else {
         return [
-          this.$t('Settings.General Settings.Preferred API Backend.Invidious API')
+          this.$t('Settings.General Settings.Preferred API Backend.Invidious API'),
+          this.$t('Settings.General Settings.Preferred API Backend.Piped API')
         ]
```

Comment:
**suggestion:** Adjust ordering to be Invidious -> Piped -> Local, so that non-local backends are grouped

Reply:
This last one isn't a major concern, just want your response at least. @ChunkyProgrammer

Reply:
Oops, forgot to respond. I think I put piped at the bottom because the piped integration isnt complete yet or i might have done it so it'd be in alphabetical order